tests/integration/helpers.py

```diff
@@ -546,7 +546,20 @@ async def dispatch_workflow(
     """
     start_time = datetime.now(timezone.utc)
 
+    # TODO: Debug
```

Comment:
Should we remove this TODO (if we implement strict linting this would throw an error)?